all-repos.yaml

```diff
@@ -242,3 +242,5 @@
 - https://github.com/igrr/astyle_py
 - https://gitlab.com/codementors/pre-commit/add-issue-number-to-conventional-commit
 - https://github.com/numpy/numpydoc
+- https://github.com/c0m1c5an5/gitlab-ci-tools
+- https://github.com/c0m1c5an5/codeowners-generator
```

Comment:
`require_serial` does not guarantee that all files are passed in a single invocation so I don't think this hook is correct